src/Main.tsx

```diff
@@ -68,8 +36,7 @@ export class Container extends React.Component<Properties> {
             </FeatureFlag>
```

Comment:
There's some additional clean-up I'd like to do here. I would like to move all of the children to `messenger-main.tsx`, and delete this file. In my opinion, it's just creating clutter now 🧹

I would also like to move `messenger-main.tsx` to `apps/messenger/index.tsx`. In general, I think storing all app entry points in `apps/***` will keep things tidy.

I can do this in a follow-up PR if y'all agree with the idea 🕵️‍♀️